src/persist/src/indexed/cache.rs

```diff
@@ -96,7 +96,10 @@ impl<B: BlobRead> BlobCache<B> {
     /// This method is idempotent. Returns true if the blob had not
     /// previously been closed.
     pub fn close(&mut self) -> Result<bool, Error> {
-        block_on(self.blob.lock()?.close())
+        let async_guard = self.async_runtime.enter();
```

Comment:
We missed those when the others were fixed?

Reply:
Yeah, I had them at one point while writing the PR, but I think I lost them in a bad rebase or something